extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -287,6 +300,10 @@ describes.realWin(
             },
             'consentState': 'accepted',
             'consentStateValue': 'accepted',
+            'consentMetadata': constructMetadata(
```

Comment:
nit: do you mind rewrite the test, to only check the `promptTrigger` value? Thanks!